README.md

```diff
@@ -148,17 +150,42 @@ kubectl create -n <ns> -f config/samples/cluster_v1beta1_backup.yaml
 
 # Testing
 
-Example of a Backup execution with a backup in progress
+## Backup resources
+
+After you create a `backup.cluster.open-cluster-management.io` resource you should be able to run `oc get cbkp -A` and get the status of the backup executions.
+
+In the example below, the last completed backup is `backup-acm-2021-08-10-140404` and a new backup is currently running `backup-acm-2021-08-10-151345`.
+The name of the backup follows this template: `acm-backup-<timestamp>`
+
+Example of a `backup.cluster.open-cluster-management.io` execution with a `velero.io.backup` in progress:
 
 ```
 $ oc get cbkp -A
 NAMESPACE       NAME         PHASE        BACKUP                         LASTBACKUP                     LASTBACKUPTIME         DURATION   MESSAGE
 oadp-operator   backup-acm   InProgress   backup-acm-2021-08-10-151345   backup-acm-2021-08-10-140404   2021-08-10T18:22:07Z   18m3s      Velero Backup [backup-acm-2021-08-10-151345] phase:InProgress ItemsBackedUp[439], TotalItems[1410]
 ```
-Example of a Backup execution with no backup in progress
+
+Running the command below returns all `velero.io.backup` resources.
+You should also be able to see the backup files under the storage location defined when installing the OADP Operator.
+
+```
+$ oc get backup -A
```

Comment:
As far as I understand, I would replace this command with `oc get backup -n oadp-operator`. Today the backup/restore operator create and consumes resources exclusively in the namespace where velero installation is running. For us it's `oadp-operator`.

Reply:
That's a good point ! I am going to change it with 
```
Running the command below returns all velero.io.backup resources, created in the same namespace with the OADP Operator. Replace the <oadp_operator_ns> with the namespace name used to install the OADP Operator (the default value for the OADP Operator install namespace is oadp_operator).

You should also be able to see the backup files under the storage location defined when installing the OADP Operator.

$ oc get backup -n <oadp_operator_ns>

```